.stylelintignore

```diff
@@ -0,0 +1,3 @@
+libs/wikipedia-preview-link.css
+src/**/*.js
+src/**/*.svg
```

Comment:
We need a newline at the end of this file

Reply:
Done